ui/src/Components/CreateProfile.jsx

```diff
@@ -11,30 +11,119 @@ const H3 = styled.h3`
   margin-left: 50px;
 `;
 
-const CreateProfile = () => {
-    return (
-        <React.Fragment>
-            <NavBar />
-            <div>
-                <H3>Complete Profile</H3>
-            </div>
-            <Form>
+class CreateProfile extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            firstName: '',
+            lastName: '',
+            email: '',
+            number: '',
+            profile_id: '',
+            ref1Name: '',
+            ref1Email: '',
+            ref2Name: '',
+            ref2Email: '',
+        };
+
+        this.handleFirstNameChange = this.handleFirstNameChange.bind(this);
+        this.handleLastNameChange = this.handleLastNameChange.bind(this);
+        this.handleEmailChange = this.handleEmailChange.bind(this);
+        this.handleNumChange = this.handleNumChange.bind(this);
+        this.handleRef1NameChange = this.handleRef1NameChange.bind(this);
+        this.handleRef1EmailChange = this.handleRef1EmailChange.bind(this);
+        this.handleRef2NameChange = this.handleRef2NameChange.bind(this);
+        this.handleRef1EmailChange = this.handleRef2EmailChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+
+    }
+
+    handleFirstNameChange(event) {
+        this.setState({firstName: event.target.value});
+    }
+
+    handleLastNameChange(event) {
+        this.setState({lastName: event.target.value});
+    }
+
+    handleEmailChange(event) {
+        this.setState({email: event.target.value});
+    }
+
+    handleNumChange(event) {
+        this.setState({number: event.target.value});
+    }
+
+    handleRef1NameChange(event) {
+        this.setState({ref1Name: event.target.value});
+    }
+
+    handleRef1EmailChange(event) {
+        this.setState({ref1Email: event.target.value});
+    }
+
+    handleRef2NameChange(event) {
+        this.setState({ref2Name: event.target.value});
+    }
+
+    handleRef2EmailChange(event) {
+        this.setState({ref2Email: event.target.value});
+    }
+
+    handleSubmit(event) {
+        event.preventDefault();
+        const resp = fetch( 'https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/insertFormData',
```

Comment:
We should have a global config for the URL too and then just add the endpoint, considering we're using it in so many places. Would also make it a lot easier to switch bw local and cloud development.